src/main/java/com/javatemplate/api/book/BookController.java

```diff
@@ -23,4 +22,34 @@ public class BookController {
     public List<BookDTO> findAll() {
         return toBookDTOs(bookService.findAll());
     }
+
+    @Operation(summary = "Find by id")
+    @GetMapping("{bookId}")
+    public BookDTO findById(final @PathVariable UUID bookId) {
+        return toBookDTO(bookService.findById(bookId));
+    }
+
+    @Operation(summary = "Find books by name, author and description")
+    @GetMapping("search")
+    public List<BookDTO> search(final @RequestParam String searchTerm) {
+        return toBookDTOs(bookService.findByNameAuthorDescription(searchTerm));
```

Comment:
```suggestion
        return toBookDTOs(bookService.find(searchTerm));
```

find với search là đồng nghĩa nên chỉ dùng 1 thôi

Reply:
Fixed